code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -207,3 +207,18 @@
 	tools = list(/obj/item/weapon/wirecutters)
 
 	time = 40
+
+/obj/item/clothing/shoes/medsandal

```

Comment:
Just make it a child like /obj/item/clothing/shoes/sandal/med and /obj/item/clothing/shoes/sandal/fancy and it'll be :moneybag: